### ISSUE TITLE
Protocol "https:" not supported. Expected "http:"

### ISSUE THREAD

**Issue by BlueHatbRit** [f/graphql.js#10]
When trying to access the GitHub GraphQL API I'm getting the following error:

```
Error: Protocol "https:" not supported. Expected "http:"
    at new ClientRequest (_http_client.js:55:11)
    at Object.exports.request (http.js:31:10)
    at __request (/Users/elliotblackburn/code/life-bot/node_modules/graphql.js/graphql.js:51:22)
    at /Users/elliotblackburn/code/life-bot/node_modules/graphql.js/graphql.js:245:11
    at caller (/Users/elliotblackburn/code/life-bot/node_modules/graphql.js/graphql.js:244:16)
    at Object.<anonymous> (/Users/elliotblackburn/code/life-bot/scratch.js:44:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
```

Are you able to enable https for the module?

**Comment by tfiwm** [comment]
https://github.com/f/graphql.js/pull/12

**Comment by tfiwm** [comment]
@BlueHatbRit Was fixed with the new release 0.4.11

**Comment by BlueHatbRit** [comment]
Thanks, closing the issue
